PIC_undo_batch.py

```diff
@@ -119,40 +128,42 @@ def picturae_csv_undo(self, table: str, MD5: str):
         """
         md5_start = f'''SELECT StartTimeStamp FROM {table} WHERE batch_MD5 = "{MD5}";'''
 
-        start_time = self.batch_db_connection.get_one_record(md5_start)
+        start_time = str(self.batch_db_connection.get_one_record(md5_start))
 
         md5_end = f'''SELECT EndTimeStamp FROM {table} WHERE batch_MD5 = "{MD5}";'''
 
-        end_time = self.batch_db_connection.get_one_record(md5_end)
+        end_time = str(self.batch_db_connection.get_one_record(md5_end))
 
         if start_time is None or end_time is None:
             raise ValueError(f"{MD5} not found in database table")
 
-        time_stamp_list = []
 
-        time_stamp_list.append(str(start_time))
-        time_stamp_list.append(str(end_time))
+        attachment_locations = self.get_attachment_location(timestamp1=start_time, timestamp2=end_time)
+
+        for attachment in attachment_locations:
+            print(attachment)
+            self.image_client.delete_from_image_server(attach_loc=attachment, collection='Botany')
```

Comment:
removing attachments using image_client.delete_from_image_server, which sends an api command. Might want to change the collection term into a config term.